model/api/src/main/java/org/keycloak/models/UserCredentialModel.java

```diff
@@ -8,6 +8,7 @@
  */
 public class UserCredentialModel {
     public static final String PASSWORD = "password";
+    public static final String PASSWORD_EXPIRED = "password-expired";

```

Comment:
"password-history"  makes more sense here. But, for all other message literals or error literals or UI strings, IMHO, its better to keep term as expired rather than history.
eg.
public static final String INVALID_EXPIRED_PASSWORD = "invalidExpiredPasswordMessage";